### PR TITLE
[STORM-2622] Add owner resource summary on storm UI

### PR DESCRIPTION
**Pull request by Ethanlm**
At Yahoo, we have a "Owner Summary" section in Storm UI to display the resources a user is using and guaranteed. This work is done by @abellina. I am porting the code back to apache storm.

https://issues.apache.org/jira/browse/STORM-2622

Fixed some storm-cassandra checkstyle isses to pass the build.

### CODE REVIEW

File: storm-server/src/main/java/org/apache/storm/LocalCluster.java [hunk 1]
```diff
@@ -1095,6 +1096,12 @@ public TopologyHistoryInfo getTopologyHistory(String user) throws AuthorizationE
             return ret;
         }
     }
+
+    @Override
+    public List<OwnerResourceSummary> getOwnerResourceSummaries(String owner) throws AuthorizationException, TException {
+        // TODO Auto-generated method stub
+        throw new RuntimeException("NOT IMPLMENETED YET");
```
Comment (HeartSaVioR) [inline storm-server/src/main/java/org/apache/storm/LocalCluster.java]: nit: `NOT IMPLEMENTED YET`

**Comment by HeartSaVioR** [comment]
@Ethanlm 
Could you share the screenshot of the changed page? That helps to see the benefit of the patch easily.

File: storm-server/src/main/java/org/apache/storm/LocalCluster.java (hunk 1 above)
Comment (Ethanlm) [inline storm-server/src/main/java/org/apache/storm/LocalCluster.java]: Thanks. Fixed it.

**Comment by Ethanlm** [comment]
screen shot 2017-07-14 at 9 08 15 am
screen shot 2017-07-14 at 9 09 08 am

**Comment by vesense** [comment]
Looks great. +1

**Comment by Ethanlm** [comment]
Added some unit tests

**Comment by harshach** [comment]
+1

**Pull request merged**